handlers/sf-api-user-credentials-setter/src/main/scala/com/gu/sfapiusercredentialsetter/Config.scala

```diff
@@ -0,0 +1,16 @@
+package com.gu.sfapiusercredentialsetter
+
+case class Config(salesforceConfig: SalesforceConfig, awsConfig: AwsConfig)
+
+case class SalesforceConfig(
+  authUrl: String,
+  clientId: String,
+  clientSecret: String,
+  userName: String,
+  password: String,
+  token: String
+)
+
+case class AwsConfig(
+  environment: String
```

Comment:
perhaps this would be better, as environment usually means a set of values from the environment whereas everyone knows what `stage` means:
```suggestion
  stage: String
```

Reply:
I've updated to use 'stage' terminology rather than 'environment'